dynawo/sources/Models/CPP/CMakeLists.txt

```diff
@@ -6,12 +6,13 @@
 # file, you can obtain one at http://mozilla.org/MPL/2.0/.
 # SPDX-License-Identifier: MPL-2.0
 #
-# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+# This file is part of Dynawo, a hybrid C++/Modelica open source time domain simulation tool for power systems.
```

Comment:
Please do not modify the copyright of existing files

Reply:
:'(